### ISSUE TITLE
Move docs to gitbook

### ISSUE THREAD

**Issue by adlerjohn** [FuelLabs/docs#84]
Instead of our custom build for docs, it might be worth looking into a standard platform for docs, like gitbook.

**Comment by SilentCicero** [comment]
Agreed, although there is some nice benefits to having a custom system.
